lib/farm.js

```diff
@@ -57,6 +58,10 @@ Farm.prototype.setup = function (methods) {
   this.activeChildren = 0
   this.callQueue      = []
 
+  if (this.options.autoStart)
```

Comment:
minor style nit: put braces around this one, my general rule is that if you have >1 line under an `if`, or the related `else`, then brace everything even if you don't technically need it; just for reading clarity